README.md

```diff
@@ -13,213 +13,56 @@ can also be accessed at:
 
-Most of our team uses Visual Studio Code (VS Code) to write code. 
-If you use VS Code, here are a few extensions that might be helpful.
+### Build Warnings
+You may see the following harmless warnings when building.
 
-- This extension provides full support for editing ReasonML source code and
-  relevant tools:
-
-  - [ocaml-platform](https://github.com/ocamllabs/vscode-ocaml-platform)
-
-- Due to Reason's poor parse errors, unbalanced parentheses can be difficult
-  to find.  The following extensions help with that.
-
-  - [Bracket Pair Colorizer 2](https://marketplace.visualstudio.com/items?itemName=coenraads.bracket-pair-colorizer-2)
-  - [Indenticator](https://marketplace.visualstudio.com/items?itemName=sirtori.indenticator)
-  - [indent-rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow)
-
-In addition to these extensions, enabling the breadcrumbs bar can make
-navigating a large code base easier. There are multiple ways to make the
-breadcrumbs bar visible:
-
-- Click **View** / **Show Breadcrumbs** from the menu bar.
-- Press `Ctrl+Shift+P` (macOS: `Cmd+Shift+P`), start typing `breadcrumbs`, and
-  select `View: Toggle Breadcrumbs` from the dropdown menu to toggle breadcrumbs
-  on and off.
-- Press `Ctrl+Shift+.` to start breadcrumbs navigation.
-
-### Suggested Setup for NeoVim
-
-If you enjoy your Vim binding and Vim setup, the following may help you set up your Reason IDE in NeoVim.
-
-If you use vim, I recommend you to switch to NeoVim since it has a better support for multi-thread,
-and thus less likely to block you when you are programming.
-
-To set up the LSP (Language Server Protocol), you need to set up your Language Client for Neovim and Language Server for ocaml.
-- [ocaml-language-server](https://www.npmjs.com/package/ocaml-language-server)
-- [LanguageClient-neovim](https://github.com/autozimu/LanguageClient-neovim)
-
-After installing the previous two, you may want to copy the following to your neovim config file. 
-(assuming `npm` have ocaml-language-server installed under `/usr/bin`)
+#### Ezjs_idb 
 ```
-let g:LanguageClient_serverCommands = {
-    \ 'ocaml': ['/usr/bin/ocaml-language-server', '--stdio'],
-    \ 'reason': ['/usr/bin/ocaml-language-server', '--stdio']
-    \ }
-" LanguageClient-neovim
-nnoremap <F5> :call LanguageClient_contextMenu()<CR>
-" Or map each action separately
-nnoremap <silent> K :call LanguageClient#textDocument_hover()<CR>
-nnoremap <silent> gd :call LanguageClient#textDocument_definition()<CR>
-nnoremap <silent> gr :call LanguageClient#textDocument_references()<CR>
-nnoremap <silent> gf :call LanguageClient#textDocument_formatting()<cr>
-nnoremap <silent> <F2> :call LanguageClient#textDocument_rename()<CR>
+Warning 58 [no-cmx-file]: no cmx file was found in path for module Ezjs_idb, and its interface was not compiled with -opaque
```

Comment:
is this still happening? don't see it anymore. if not, we can drop this bit of the README and close #1104

Reply:
We silenced the warning so it's not appearing amymore. Whatever the warning means still exists. Not sure if you want to  remove the issue.